TradeItIosTicketSDK2/TradeItPreviewBrandedAccountNameCell.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+class TradeItPreviewBrandedAccountNameCell: UITableViewCell {
+    @IBOutlet weak var accountName: UILabel!
+    @IBOutlet weak var logo: UIImageView!
+    
+    override func awakeFromNib() {
```

Comment:
I'm not sure what this method is doing. I copied from another branded cell.

Reply:
You won't need to add the accessory view. It's the `>` arrow on the right. You will need to apply the theme configurator here though. Look at some other cells for examples.